src/test/java/com/example/project/page_object/ProductPage.java

```diff
@@ -0,0 +1,45 @@
+package com.example.project.page_object;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class ProductPage {
+    private WebDriver driver;
+
+    @FindBy(xpath = "//h1[@class='product-name cms-ae-product-name']")
```

Comment:
I recall that earlier I told to stick to the same locators, but in case of class based ones it would be neater to use CSS. And other thing to note is that it is not needed to specify both classes. In case one class uniquely identifies the element then only it can be used.